overthebox/defaults/network.defaults

```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
 EOF
 fi
 
+uci get network.globals.tcp_congestion_control >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+  uci -q batch <<-EOF >/dev/null
+	set network.globals=globals
+	set network.globals.tcp_congestion_control='cubic'
```

Comment:
`$DEFAULT_TCP_CONGESTION_CONTROL` instead of `cubic` here to if you agree with my first comment